velodrome/fetcher/client.go

```diff
@@ -119,7 +119,7 @@ func (client *Client) limitsCheckAndWait() {
 type ClientInterface interface {
 	RepositoryName() string
 	FetchIssues(time.Time, chan *github.Issue)
-	FetchIssueEvents(*int, chan *github.IssueEvent)
+	FetchIssueEvents(int, *int, chan *github.IssueEvent)
```

Comment:
wouldn't this interface be easier to read if it specified the meaning of each arg? aka `FetchIssueEvents(issueID int)` rather than just `FetchIssueEvents(int)`